packages/tasks/src/tasks/audio-classification/inference.ts

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
In term of repo structure, I would put all the `.ts` generated files inside a `./generated` folder to make it explicit to any reader that the files are generated.

(typically comments from @osanseviero should be made on the `input.json`/`output.json` files instead of `inference.ts`).

Reply:
Hmm, I kindof like the fact that you can import types from `tasks/text-generation/inference` directly, and that the inference types are close to the JSON Schema spec